velox/exec/tests/utils/HiveConnectorTestBase.cpp

```diff
@@ -103,6 +103,33 @@ void HiveConnectorTestBase::writeToFile(
   writer.close();
 }
 
+void HiveConnectorTestBase::writeToFileCommon(
```

Comment:
This comment is for line 89
It's still using velox::dwrf::WriterOptions options;
Will you be able to change it to the common one?

Reply:
Remove #include "velox/common/file/FileSystems.h"
it's not used